clients/benchmarks/CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ target_include_directories( client
     $<BUILD_INTERFACE:${Boost_INCLUDE_DIRS}>
 )
 
+link_libraries( ${BOOST_ROOT}/lib )
```

Comment:
I don't think you need to use link_libraries()?  ${Boost_LIBRARIES} should be absolute paths.  Typically, explicitly calling link_libraries is a bad idea.